src/main/scala/uk/gov/hmrc/play/http/HeaderCarrier.scala

```diff
@@ -114,10 +120,25 @@ object HeaderCarrier {
       headers.get(HeaderNames.trueClientPort),
       headers.get(HeaderNames.googleAnalyticTokenId),
       headers.get(HeaderNames.googleAnalyticUserId),
-      getDeviceId(cookies, headers)
+      getDeviceId(cookies, headers),
+      headers.get(HeaderNames.akamaiReputation).map(AkamaiReputation),
+      remainingHeaders(headers)
     )
   }
 
+  val blacklistedHeaders: Seq[String] = {
+    // TODO sensible config key name and default values
```

Comment:
I'd probably call it 'httpHeadersBlacklist' (present tense rather than past) but to be honest your name is fine really. Can just remove the TODO if you're happy with the defaults.